### ISSUE TITLE
Unrecognized font family: 'body'

### ISSUE THREAD

**Issue by konstantinjdobler** [GeekyAnts/NativeBase#4500]
I have a React Native CLI project and when using Native Base `<Text>` components with a `fontFamily="body"`, I am getting a `Unrecognized font family: 'body'` error. I have not modified the fonts in my theme. 

![Simulator Screen Shot - iPhone 13 - 2022-01-06 at 19 39 39](https://user-images.githubusercontent.com/28780372/148434042-3383af17-284e-474b-bf07-140558da65a3.png)

NativeBase version: 3.3.0

**Comment by ankit-tailor** [comment]
Hi @konstantinjdobler, Can you please provide snack / codesandbox link?

**Comment by konstantinjdobler** [comment]
I can't get native-base running in codesandbox because of this error:
```js
(0,_reactNative.requireNativeComponent) is not a function. 
(In '(0,_reactNative.requireNativeComponent)('RNSVGSvgView')', 
'(0,_reactNative.requireNativeComponent)' is undefined)
```

But I tracked down the source of the error to this problem: It occurs when nesting `<Text>` components.
This produces the error:
```jsx
<Text fontFamily="body">
  Lorem
  <Text fontFamily="body" bold>Ipsum</Text>
</Text>
```

This is fine:
```jsx
<Text fontFamily="body">
  Lorem
  <Text bold>Ipsum</Text>
</Text>
```

Now there doesn't seem to be a *need* to specify the same `fontFamily` in the nested scenario, but the error message really doesn't make it clear what the actual cause is (you don't even get a problematic line of code to look into). Given that we spent quite a lot of time on this issue, it might also happen to others. Fixing this or providing a clear error message is probably desirable. @ankit-tailor

**Comment by ankit-tailor** [comment]
Hi @konstantinjdobler , sure, we'll look into this.

**Comment by surajahmed** [comment]
This issue has been fixed in version 3.3.6.

**Comment by dkulakov** [comment]
I have the same issue in 3.4.6 & 3.4.8
